### DISCUSSION TITLE
.NET 5 runtime and Windows

### DISCUSSION THREAD

**Discussion by asdfCYBER** [dotnet/runtime#51003] [Q&A] [answered]
Hi all,

Could you please tell me if .NET 5 runtime will be shipped with Windows in a future update like .NET Framework? I found this answer on a question from December, but I don't know if it still applies: https://github.com/dotnet/runtime/discussions/46363#discussioncomment-238231. Since .NET 5 has superseded .NET Framework as far as I understand, I hoped runtimes would continue to be shipped with Windows.

**Comment by stephentoub** [comment, marked as answer]
We do not plan to do so.

**Reply by asdfCYBER** [reply]
Thanks for the quick answer. I suppose including the runtime installer with my applications will be the next best thing then.

**Reply by GeraudFabien** [reply]
Depending on you're situation this may help https://devblogs.microsoft.com/dotnet/net-core-updates-coming-to-microsoft-update/
But for other case you also can do like .Net framework where the installer check if install and install it if not. It was how i used to  work on .Net framework because you can't know the version install on the client.

**Reply by asdfCYBER** [reply]
@GeraudFabien yes I saw your answer on the question I linked, automatic updates are very nice because I won't have to update the version like I would if I went with a self-contained solution. I still have to install a runtime first though. My application is portable and doesn't have an installer, so I will let users install it manually.
